### ISSUE TITLE
Noisy with bleadperl

### ISSUE THREAD

**Issue by andk** [kasei/perl-iri#9]
There was a short discussion on perl5-porters about a new warning in bleadperl that impacts IRI:

http://www.gossamer-threads.com/lists/perl/porters/328912?page=last

A typical cpantesters report for IRI with bleadperl nowadays is cut off:

http://www.cpantesters.org/cpan/report/4eb0c846-4691-11e6-8013-f1fd5bc2a771

The impact is huge for cpantesters as a whole because all dependent modules also produce heaps of warnings and lead to more cut off reports.

I would be quite relieved if a new release of IRI could be made so that to get this part of cpantesters smoking back to normal.

Thanks!

**Comment by kasei** [comment]
Hey @andk. Sorry. I didn't realize this was having such an impact on the testing infrastructure. I'd already merged a PR fixing the issue, but had been trying to juggle some other outstanding issues before doing a release. Hearing this, though, I decided to cherry-pick the PR and push a release just to address the regex issue. I've already uploaded a beta to CPAN, and I'll try to get a full release out in the next day or two.
